src/org/zaproxy/zap/extension/pscanrulesBeta/CookieLooselyScopedScanner.java

```diff
@@ -136,7 +136,7 @@ private void raiseAlert(HttpMessage msg, int id, String host, List<HttpCookie> l
 		}		
 		
 		alert.setDetail(getDescriptionMessage(), msg.getRequestHeader()
-				.getURI().toString(), null, getExploitMessage(), 
+				.getURI().toString(), null, "", 

```

Comment:
I kept it so that it may be used in future, if required. If you want I will remove it.